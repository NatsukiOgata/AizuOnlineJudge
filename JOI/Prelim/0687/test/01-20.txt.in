100
473 732 960 898 1825 830 525 738 1230 80 184 112 1231 726 635 1540 306 1972 201 1701 320 1935 1080 388 1077 1165 1070 741 1133 171 1382 1488 74 277 40 238 106 804 385 265 118 31 1458 1468 1296 358 63 1466 1415 1640 2000 1265 1515 939 1253 1903 655 961 1806 1586 893 367 819 1828 924 371 1572 136 730 1646 980 1346 1954 1831 984 1821 216 1770 276 1202 1619 296 450 559 1812 1303 1435 1709 197 998 783 891 1033 1675 1045 497 1352 1669 605 410
