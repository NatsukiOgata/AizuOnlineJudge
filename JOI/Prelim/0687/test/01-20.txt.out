40351
53887
